routes/movieRouter.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const movieController = require('../controllers/movieController');
+
+const router = express.Router();
+
+router.get('/', movieController.getMovies);
```

Comment:
```suggestion
router.get('', movieController.getMovies);
```
엔드포인트 끝에 `/`가 들어가는것은 REST하지 못한 엔드포인트 작성입니다. 수정해주세요~